modules/rosa-hcp/providers.tf

```diff
@@ -1 +1,16 @@
 data "aws_caller_identity" "current" {}
+
+terraform {
+  required_version = ">= 1.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 5.35.0"
+    }
+    rhcs = {
+      version = "1.6.8"
```

Comment:
for `1.6.8`, I'd recommend something like `>= ` as well otherwise any usage will be forced to keep it updated, which may not always be required.